### DISCUSSION TITLE
command palette not finding repositories

### DISCUSSION THREAD

**Discussion by Mehuge** [community/community#46860] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

<img width="990" alt="image" src="https://user-images.githubusercontent.com/99895/217834962-3f96fd9c-adb4-44b3-babf-d7a07325964f.png">

If I search for dalang (one of my repos) it doesn't find it. 
If I search for alien (another of my repos) it finds it.

<img width="983" alt="image" src="https://user-images.githubusercontent.com/99895/217835522-08e4aeff-58f1-4f9f-a16e-5be22450b640.png">

Should it be finding these or am I doing something wrong?

**Comment by SilasRodrigues19** [comment]
hey @Mehuge 

Apparently it only works within the profile itself
